### PR TITLE
Fix a typo in tidb-monitoring-api.md

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Fix a typo in tidb-monitoring-api.md

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/6829
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-monitoring-api.md [hunk 1]
```diff
@@ -5,10 +5,9 @@ aliases: ['/docs-cn/dev/tidb-monitoring-api/']
 
 # TiDB 集群监控 API
 
-TiDB 提供了以下两种接口来监控集群状态：
+TiDB 提供了以下几种接口来监控集群状态：
 
-- [状态接口](#运行状态)：通过 HTTP 接口对外汇报组件的信息。
-- [存储信息](#存储信息)：通过 HTTP 接口对外汇报表的存储信息。
+- [状态接口](#使用状态接口)：通过 HTTP 接口对外汇报组件的信息。通过状态接口，你可获取当前 TiDB Server 的[运行状态](#运行状态)以及表的[存储信息](#存储信息)。
```
Comment (TomShawn) [inline tidb-monitoring-api.md:10]: @xhebox PTAL, thanks!

**Review by xhebox — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**